app/controllers/v0/sign_in_controller.rb

```diff
@@ -232,7 +232,7 @@ def logout_get_redirect_url
     end
 
     def handle_pre_login_error(error, client_id)
-      if SignIn::Constants::ClientConfig::COOKIE_AUTH.include?(client_id)
+      if SignIn::Constants::Auth::CLIENT_IDS.include?(client_id) && client_config(client_id).cookie_auth?
```

Comment:
I don't think we need the `SignIn::Constants::Auth::CLIENT_IDS.include?(client_id)` part of this conditional